apps/filecoin-site/src/app/_utils/generateBreadcrumbsList.ts

```diff
@@ -0,0 +1,47 @@
+import type { BreadcrumbList, ListItem } from 'schema-dts'
+
+import { BASE_URL } from '@/constants/siteMetadata'
+
+type GenerateBreadcrumbListProps = {
+  path: string
+  title: string
+  includeBlogLevel?: boolean
+}
+
+export function generateBreadcrumbList({
+  path,
+  title,
+  includeBlogLevel = false,
```

Comment:
`includeBlogLevel` because of the case in blog posts:
 {"@type":"ListItem","position":1,"name":"Home","item":"https://filecoin.io/"},
        {"@type":"ListItem","position":2,"name":"Blog","item":"https://filecoin.io/blog"},
        {"@type":"ListItem","position":3,"name":"Filecoin Ecosystem Metrics Roundup"}